js/zb.js

```diff
@@ -1138,15 +1234,15 @@ module.exports = class zb extends Exchange {
         }
         return this.safeOrder ({
             'info': order,
-            'id': id,
+            'id': orderId,
             'clientOrderId': undefined,
             'timestamp': timestamp,
             'datetime': this.iso8601 (timestamp),
             'lastTradeTimestamp': undefined,
             'symbol': market['symbol'],
             'type': type,
-            'timeInForce': undefined,
-            'postOnly': undefined,
+            'timeInForce': timeInForce,
```

Comment:
Maybe set type like `'type': 'limit'; // market order is not available in ZB`, it's one less line of code